### PR TITLE
Bug 1817522 – Add dark theme support for seach button background

### PR DESCRIPTION
**Pull request by mavduevskiy**
### Screenshots
 Old | New
 --- | --- 
 |   
 |   

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/main/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1817522

### CODE REVIEW

File: fenix/app/src/main/res/layout/search_selector.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
         android:layout_width="40dp"
         android:layout_height="28dp"
         android:layout_marginHorizontal="8dp"
-        app:cardBackgroundColor="@color/photonWhite"
+        app:cardBackgroundColor="@color/fx_mobile_layer_color_2"
```
Comment (gabrielluong) [inline fenix/app/src/main/res/layout/search_selector.xml:16]: Could you double check if this behaves correctly in private theme? `fx_mobile_layer_color_2` isn't defined in private theme and we would use `?attr/layer2` to account for that. I

**Comment by mavduevskiy** [comment]
@gabrielluong 
Thanks for pointing me into the right direction! The color was `?attr/textPrimary`. [Here](https://www.figma.com/file/epu0Sj6mCPk7K282sNpHgI/Site?node-id=3754%3A50354&t=S90QCY7EWfCWresz-4) is the link. Updated the with proper values.

File: fenix/app/src/main/res/layout/search_selector.xml [hunk 2]
```diff
@@ -41,7 +41,7 @@
                 android:layout_height="6dp"
                 android:importantForAccessibility="no"
                 app:srcCompat="@drawable/ic_chevron_down_6"
-                app:tint="@color/photonBlack"
+                app:tint="?attr/textPrimary"
```
Comment (gabrielluong) [inline fenix/app/src/main/res/layout/search_selector.xml:44]: One correction here - this should be `iconPrimary`. I have the small chevron selected and in the "Colors" section, you will see Light/**Icon**/Primary. `textPrimary` should be used for text as the name implies, but **icon** primary is what you are looking for here.

Screen Shot 2023-02-22 at 5 04 16 PM

**Review by gabrielluong — APPROVED** [review]

File: fenix/app/src/main/res/layout/search_selector.xml (hunk 2 above)
Comment (mavduevskiy) [reply fenix/app/src/main/res/layout/search_selector.xml:44]: currently there is no iconPrimary attribute, and the majority is the icons in the xml are using android:fillColor="?attr/textPrimary" as a filler.

I will merge it as it is and file a bug to add the attribute and use it

**Pull request merged**